tests/case_trajectories.py

```diff
@@ -51,7 +51,7 @@ def case_nyquist_radial3D(self, Nc=32 * 4, Ns=16, Nr=32 * 4, N=32):
         trajectory = rotate(trajectory, nb_rotations=Nr)
         return trajectory, (N, N, N)
 
-    def case_nyquist_radial3D_lowmem(self, Nc=2, Ns=16, Nr=2, N=10):
+    def case_nyquist_lowmem_radial3D(self, Nc=2, Ns=16, Nr=2, N=10):
```

Comment:
This was a sidequest, the batch tests were triggering to run on this also, not like it matters, but just saving some time by changing the name

Reply:
wait, renaming the function make it run faster ?